charts/team-ns/templates/networkpolicy.yaml

```diff
@@ -111,4 +114,5 @@ spec:
 {{- end }} {{/* if ingressPrivateMode */}}
 {{- end }} {{/* not $s.isCore */}}
 {{- end }} {{/* range services */}}
+{{- end }} {{/* if "networkPolicy" "ingressPrivate" true $v */}}
 {{- end }} {{/* if $v.teamId */}}
```

Comment:
same here, indenting is better

Reply:
yea, I have different opinion about this topic. Here I do not need to scroll whole file in order to find out the block that a given `end` is closing

Reply:
you get lost with indented code? I just put my mouse cursor and scroll content...super easy

Reply:
pointless conversation

Reply:
I ask helm community about coding style: https://github.com/helm/helm/issues/10702
If there is a tool that can support it then we can go for indentation, otherwise it is tedious work that does not help me.